adagrams/game.py

```diff
@@ -1,13 +1,76 @@
+import random
 from random import randint
 
 def draw_letters():
-    pass
+    letter = []
+    pool = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", 
+        "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    pool_dic= {"A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, 
+    "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, 
+    "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1}
+
+    while len(letter) < 10:
+        index = randint(0,len(pool)-1)
+        pick_letter = pool[index]
+        if pool_dic[pick_letter]>0: 
+            letter.append(pick_letter)
+            pool_dic[pick_letter] -= 1
+        else: 
+            print("Please pick again.")
+    return letter
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_copy =[letter.lower() for letter in letter_bank]
+    word = word.lower()
+    for letter in word:
+        if letter in letter_bank_copy:
+            letter_bank_copy.remove(letter)
+        else:
+            return False
+    return True
+
 
 def score_word(word):
-    pass
+    points =  {"A": 1, "E": 1, "I": 1, "O": 1, "U": 1, "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
+    "D": 2, "G": 2,
+    "B": 3, "C": 3, "M": 3, "P": 3,
+    "F": 4, "H": 4, "V": 4, "W": 4, "Y": 4,
+    "K": 5,
+    "J": 8, "X": 8,
+    "Q": 10, "Z": 10}
+    word = word.upper()
+    if len(word)in (7,8,9,10):
```

Comment:
Though this is functionally sound, the conventional way to check if a length is a certain size is to use one the comparison operators. In this case I would do it like so: 
```suggestion
    if len(word) > 6:
```
This also more explicitly communicates the intention behind this line of code.